appengine-managed-runtime/src/main/java/com/google/apphosting/utils/config/AppEngineWebXmlProcessor.java

```diff
@@ -631,14 +631,6 @@ private boolean toBoolean(String value) {
     return (value.equalsIgnoreCase("true") || value.equals("1"));
   }
 
-  private String getTextNode(Element node) {
```

Comment:
This is a bit different than `XmlUtils.getText` as the latter is calling `getTextContent` which will also include the child nodes content (not sure if that is desired or would never happen). If that is fine lets add it to the change description.